package.json

```diff
@@ -21,7 +21,7 @@
     "build": "turbo run build",
     "changeset": "changeset",
     "changeset:publish": "pnpm run prepack && changeset publish",
```

Comment:
```suggestion
    "changeset:publish": "pnpm run prepack && changeset publish && pnpm run jsr:dry-run && pnpm run jsr:publish",
```

How about this?

Reply:
> Does this mean that JSR publishing will happen automatically when a changeset PR is merged, without needing a separate JSR Publish CI?

Yes

Reply:
https://github.com/toss/suspensive/pull/1481/commits/45bf1da7e781e0bb5e08a138b54f6a5212cb1484